### PR TITLE
Improved logs

### PR DESCRIPTION
**Pull request by esuwu**

### CODE REVIEW

**Comment by esuwu** [comment]
https://github.com/wavesplatform/nodemon/issues/71

File: pkg/api/api.go [hunk 1]
```diff
@@ -157,21 +158,21 @@
...
+		a.zap.Error("Failed to marshal statements", zap.Error(err))
 		http.Error(w, fmt.Sprintf("Failed to marshal statements to JSON: %v", err), http.StatusInternalServerError)
 		return
 	}
 	w.WriteHeader(http.StatusOK)
 
 	sumhash := strings.Replace(statehash.SumHash.String(), "\n", "", -1)
 	sumhash = strings.Replace(sumhash, "\r", "", -1)
-	log.Printf("Statement for node %s has been put into the storage, height %d, statehash %s\n", escapedNodeName, statement.Height-1, sumhash)
+	a.zap.Info(fmt.Sprintf("Statement for node %s has been put into the storage, height %d, statehash %s\n", escapedNodeName, statement.Height, sumhash))
```
[the remark this answers is not in the document]
Comment (nickeskov) [reply pkg/api/api.go:175]: Please, fix this issue.
```go
nodeName := r.Header.Get("node-name")
escapedNodeName := strings.Replace(nodeName, "\n", "", -1)
escapedNodeName = strings.Replace(escapedNodeName, "\r", "", -1)
```

[the remark this answers is not in the document]
Comment (nickeskov) [reply pkg/api/api.go:175]: Issue has been fixed, my bad.

**Review by nickeskov — CHANGES_REQUESTED** [review]
@esuwu Please, use zap in all places instead of standard logger.

```shell
$ grep -rn "log\." pkg cmd | grep -v "test.go"
pkg/storing/nodes/nodes.go:93:          log.Printf("New node #%d at '%s' was stored", id, url)
pkg/storing/nodes/nodes.go:111:         log.Printf("New node #%d at '%s' was stored", id, url)
pkg/storing/nodes/nodes.go:131:                 log.Printf("Node #%d at '%s' was deleted", id, url)
cmd/bots/discord/discord.go:28:                 log.Println(err)
cmd/bots/discord/discord.go:37:         log.Fatalf("can't initialize zap logger: %v", err)
cmd/bots/discord/discord.go:42:                 log.Println(err)
cmd/bots/internal/common/init/init.go:30:       log.Printf("telegram chat id for sending alerts is %d", chatID)
cmd/bots/internal/common/init/init.go:45:       log.Printf("discord chat id for sending alerts is %s", chatID)
cmd/bots/telegram/telegram.go:28:                       log.Println(err)
cmd/bots/telegram/telegram.go:37:               log.Fatalf("can't initialize zap logger: %v", err)
cmd/bots/telegram/telegram.go:42:                       log.Println(err)
cmd/nodemon/nodemon.go:44:                      log.Println(err)
cmd/nodemon/nodemon.go:53:              log.Fatalf("can't initialize zap logger: %v", err)
cmd/nodemon/nodemon.go:58:                      log.Println(err)
```

**Review by nickeskov — APPROVED** [review]

**Pull request merged**